x-pack/plugin/fleet/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/fleet/20_wait_for_checkpoints.yml

```diff
@@ -0,0 +1,79 @@
+setup:
+  - do:
+      indices.create:
+        index: test-refresh-disabled
+        body:
+          settings:
+            number_of_shards: "1"
+            refresh_interval: "-1"
+
+  - do:
+      indices.create:
+        index: test-after-refresh
+        body:
+          settings:
+            number_of_shards: "1"
+
+  - do:
+      indices.put_alias:
+        index: test-after-refresh
+        name: test-alias
+
+  - do:
+      index:
+        index: test-timeout
```

Comment:
This index is unused. I wonder if you wanted to add a test with an index with a long timeout? Or perhaps that cannot be done reliably here?

Reply:
I used to have a test. But once we added the functionality that the document has to be indexing in order to wait on checkpoint, it became very difficult to test reliably.